spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/SvnKitEnvironmentRepository.java

```diff
@@ -179,7 +179,15 @@ protected File getWorkingDirectory() {
 
 	private File getSvnPath(File workingDirectory, String label) {
 		// use label as path relative to repository root
-		return new File(workingDirectory, label);
+		// if it doesn't exists check branches and then tags folders
+		File svnPath = new File(workingDirectory, label);
+		if(!svnPath.exists()) {
+			svnPath = new File(workingDirectory, "branches" + File.separator + label);
+		}
+		if(!svnPath.exists()) {

```

Comment:
I'm not sure what your asking.  It checks the root folder for the label, if it doesn't exists it checks the branches folder for the label, if label doesn't exists there it checks the tags folder for the label.